react-list.es6

```diff
@@ -86,7 +86,7 @@ export default class extends Component {
 
   componentWillUnmount() {
     window.removeEventListener('resize', this.updateFrame);
-    this.scrollParent.removeEventListener('scroll', this.updateFrame);
+    this.scrollParent && this.scrollParent.removeEventListener('scroll', this.updateFrame);
```

Comment:
I agree that this is gross. I'm going to do some more investigation tonight
and get a minimal poc together.
On Jan 12, 2016 7:15 PM, "Casey Foster" <notifications@github.com> wrote:

> In react-list.es6
> <https://github.com/orgsync/react-list/pull/82#discussion_r49540445>:
>
> > @@ -86,7 +86,7 @@ export default class extends Component {
> >
> >    componentWillUnmount() {
> >      window.removeEventListener('resize', this.updateFrame);
> > -    this.scrollParent.removeEventListener('scroll', this.updateFrame);
> > +    this.scrollParent && this.scrollParent.removeEventListener('scroll', this.updateFrame);
>
> Have you found an instance where this is null? That would mean
> componentWillUnmount was called before componentDidMount, as
> componentDidMount is where this.scrollParent is set.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/orgsync/react-list/pull/82/files#r49540445>.
>